src/backend/graphItemService.ts

```diff
@@ -113,6 +113,9 @@ export class GraphItemService {
       sourceKey: sourceKey
     });
     if (!queryResponse.isSuccess()) {
+      if (queryResponse.body.key === LkErrorKey.FORBIDDEN) {
+        throw Error(`Access error, you are not allowed to execute queries on data-source "${sourceKey}", please use an admin account along with a partial schema.`);
```

Comment:
Could a "Read & Run Queries" user fall into this ? If the answer is yes it's not appropriate. If the answer is no it is appropriate.

Reply:
Yes, a "Read & Run Queries" will fall Into this

Reply:
execute **raw** queries ?

Reply:
@davidrapin any idea of a better wording ?

Reply:
Or perhaps stay super simple "Access error, please use an admin account along with a partial schema."

Reply:
@XavierDeMontis done

Reply:
“along partial schema” doesn’t sounds super clear.
Maybe:
“Access error: please make sure strict schema is disabled and you are using an admin account”